cmake/cpackEngineDEB.cmake

```diff
@@ -56,11 +56,11 @@ if (EXISTS "/etc/debian_version")
     set(DEBIAN_VERSION_NUMBER "${CMAKE_MATCH_1}")
 endif ()
 if ("${DEBIAN_VERSION_NUMBER}" EQUAL "8")
-    SET(CPACK_DEBIAN_PLATFORM_PACKAGE_DEPENDS "openssl, file, libdbi-perl, libreadline-dev, rsync, net-tools, libboost-all-dev, libsnappy1, libjemalloc1, MariaDB-server, python3")
+    SET(CPACK_DEBIAN_PLATFORM_PACKAGE_DEPENDS "openssl, file, libdbi-perl, libreadline-dev, rsync, net-tools, libboost-all-dev, libsnappy1, MariaDB-server, python3")
```

Comment:
We don't use cpack for debian packaging since 10.5 but the server's configs that are in ./debian directory in the server repo?